### ISSUE TITLE
QA: Parkbuchen Xhain

### ISSUE THREAD

**Issue by tordans** [osmberlin/parkraum.osm-verkehrswende.org#21]

**Comment by SupaplexOSM** [comment]
@tordans Weißt du, ob MapRoulette mit Multipolygonen umgehen kann? Ich könnte nämlich den Parkbuchten eine eindeutige Kennung für das Straßensegment geben, an dem sie sich befinden. Dann bilden alle Parkbuchten am gleichen Straßenabschnitt (zwischen zwei Kreuzungen) ein Multipolygon, was nur eine gemeinsame Aufgabe für dieses Segment erzeugen würde (und nicht mehrere Aufgaben für jede einzelne Parkbucht - selbst mit Puffer würde es oft mehrere Aufgaben geben).

**Comment by tordans** [comment]
Ja, das klingt nach einer guten Lösung. Erstell gern mal ne Datei und ich teste. Konnte in der Doku von Maproulette nichts finden und sie supporten ja GeoJSON … sollte also klappen.

**Comment by SupaplexOSM** [comment]
Kannst du es mit dieser mal versuchen?

[missing_street_side_parking_friedrichshain_kreuzberg_2022-09-12.zip](https://github.com/osmberlin/parkraum.osm-verkehrswende.org/files/9551673/missing_street_side_parking_friedrichshain_kreuzberg_2022-09-12.zip)

**Comment by SupaplexOSM** [comment]
Eine Handlungsanweisung könnte etwa lauten:

1) Prüfe die Stelle mit Mapillary und/oder auf aktuellen Luftbildern, ob es (wirklich noch) Parkbuchten gibt.
2) Falls ja: Schaue idealerweise in ALKIS, ob die Parkbuchten dort erkennbar sind, mit aktuellen Luftbildern und Mapillary übereinstimmen und zeichne sie nach. Alternativ Straßenbefahrung nutzen, falls ALKIS nicht aktuell ist oder nicht eingebunden werden kann.
3) Tagge die gezeichnete Fläche mit `amenity=parking` + `parking=street_side`. Versuche, die Ausrichtung der Fahrzeuge zu erkennen und tagge diese mit `parking:orientation=parallel/diagonal/perpendicular` (parallel, schräg oder quer zur Fahrbahn).
4) Falls du auf aktuellen Mapillary-Fotos die Oberfläche der Parkbucht erkennst, ergänze nach Möglichkeit auch `surface=*` (z.B. `asphalt`, `sett`, `paving_stones`).

**Comment by tordans** [comment]
# Update

- Kampagne MapRoulette erstellt https://maproulette.org/browse/challenges/28762
- GeoJSON https://tiles.osm-berlin.org/quality-assurance/berlin-street-side-parking/missing_street_side_parking_friedrichshain_kreuzberg.geojson
  - Hinweis: GeoJSON Projection muss immer projection 4326 / WGS 84 sein um es im Web verarbeiten zu können
- Wiki-Seite erstellt https://wiki.openstreetmap.org/wiki/Berlin/Verkehrswende/Parkraum/Mapping_Kampagne_Xhain/QA-parkbuchten
- Task Beschreibung von oben übertragen ins Wiki + MapRoulette

# Neues TODO

- [ ] Ideal wäre noch für die Doku ein paar Punkte dazu, wie wir die Daten verarbeitet haben @SupaplexOSM  unter https://wiki.openstreetmap.org/wiki/Berlin/Verkehrswende/Parkraum/Mapping_Kampagne_Xhain/QA-parkbuchten

**Comment by SupaplexOSM** [comment]
@tordans wie aufwendig ist es, wie bei den Einfahrten wieder einen Mapillary-Link in die Challenge einzubauen?
(In meiner Vorstellung sind diese Challenges Blaupausen für spätere Mapping-Kampagnen anderenorts, daher würde es sich vor allem lohnen, falls man diese Challenges überhaupt auf andere Orte übertragen kann - wobei fraglich ist, wo es überhaupt externe Daten zu Parbuchten gibt und eine solche Challenge daher möglich wäre...)

**Comment by SupaplexOSM** [comment]
Ich mache hier mal zu. Die Daten für Friedrichshain-Kreuzberg sind inzwischen mehr oder weniger vollständig erhoben und bearbeitet und MapRoulette-Challenge war sehr hilfreich/kann bei Gelegenheit für andere Bezirke übernommen werden.

**Comment by tordans** [comment]
Ich schließe das Ticket.
